data/human/free worlds 0 prologue.txt

```diff
@@ -1767,7 +1765,7 @@ mission "FW Katya 5"
 			choice
 				`	"Fine, I'm in."`
 					accept
-				`	"Sorry, but I think I'm done with helping the Free Worlds. I'd rather strike out on my own."`
+				`	"I'd rather be shooting down pirates. I'm done with helping the Free Worlds."`
```

Comment:
"I'd rather be shooting down pirates" maybe fits the kind of personality that would go the adventure route, but the statement itself is a bit of a non sequitur.